docs/quickstart/install.md

```diff
@@ -153,6 +153,12 @@ above by running the following before any other CLI commands:
 
 - `tanzu config eula accept` to accept the EULA.
 
+****Customer Experience Improvement Program (CEIP) Prompt**** :
+The Tanzu CLI prompts the user to accept participation in CEIP or not.
+
```

Comment:
Can you remove lines 150 to 154 and add them below to avoid duplication?
This is how things were before commit fd054aa433c977e54aad2411827f6769310fbc54